ci/docker/spiceai-init.dockerfile

```diff
@@ -0,0 +1,43 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# Nightly image is temporarily used to include FlightSQL schema fix until this is included in the next official release 1.4.0
+# FROM spiceai/spiceai:1.4.0-models
+FROM ghcr.io/spiceai/spiceai-nightly:20250611-cd21f4a-models
+
+# Create the Spicepod configuration file
+COPY <<EOF /app/spicepod.yaml
+version: v1
+kind: Spicepod
+name: app
+
+datasets:
+  - from: s3://spiceai-demo-datasets/tpch/nation/
+    name: nation
+    params:
+      file_format: parquet
+      client_timeout: 60s
+EOF
+
+# Spice runtime start-up arguments
+CMD ["--http","0.0.0.0:8090","--metrics", "0.0.0.0:9090","--flight","0.0.0.0:50051"]
+
+EXPOSE 8090
+EXPOSE 9090
+EXPOSE 50051
+
+# Start the Spicepod
```

Comment:
Redundant comment? Or missing directive?

Reply:
Removed redundant comment  - thank you. The dockerfile has been updated to use additional image for proper healthcheck (curl dependency)